### DISCUSSION TITLE
Time Keyframe

### DISCUSSION THREAD

**Discussion by 8-8Sophie8-8** [keiwando/toonsquid#708] [Q&A] [answered]
There are no tutorial anywhere ( I have not found any that talk about this specific keyframe ) to guide me on what it can do. I try using it , testing it , nothing happened with it , so cannot see what it can do, it’s purpose . Help and thanks 
![IMG_9123-compressed](https://github.com/user-attachments/assets/1cceff44-44dd-43d9-9dde-4b1712a29f20)

**Comment by Cat-Butler** [comment, marked as answer]
Use it with symbols.

You can use it to set an **_offset_** keyframe for which frame of a multi-frame symbol to show at that point - good for cut-out style animation and part switching/swapping: select different states from a symbol e.g. select from different mouth shapes, or different hand positions, from a symbol containing multiple mouth shapes, hand positions, etc.

Most useful with hold easing, and labelled frame markers make it even easier to select a frame visually instead of using a frame number.
It is mentioned in the handbook under https://toonsquid.com/handbook/symbols/marker_animation/
